rosdep/base.yaml

```
@@ -286,7 +286,9 @@ boost:
     squeeze: [libboost1.42-all-dev]
     stretch: [libboost-all-dev]
     wheezy: [libboost-all-dev]
-  fedora: [boost-devel]
+  fedora:
```

Can this be elided to just the following?

```suggestion
  fedora: [boost-devel, boost-python2-devel]
```

---

That will break Fedora 28, where the package is called `boost-python-devel` instead (and seems to be installed by default).

We can merge this as-is, or we can wait for Fedora 28 to EOL (May 31st) and merge with your suggestion.

---

> We can merge this as-is, or we can wait for Fedora 28 to EOL (May 31st) and merge with your suggestion.

Is Fedora 28 still active with it's impending EOL? I'd be inclined to break it now if the likelihood that anyone will hit it is low.

---

As a long-time Fedora user, I have to say that the 1 year lifetime of Fedora releases means that I fall EOL more often than I'd like to admit.

That said, the rule has been wrong for all of f29 so far and no one made any noise. I'll update the rule and drop f28 support.